pkg/jsonnet/imports.go

```diff
@@ -187,16 +189,31 @@ func getSnippetHash(vm *jsonnet.VM, path, data string) (string, error) {
 	return base64.URLEncoding.EncodeToString(fullHasher.Sum(nil)), nil
 }
 
-func uniqueStringSlice(s []string) []string {
-	seen := make(map[string]struct{}, len(s))
-	j := 0
-	for _, v := range s {
-		if _, ok := seen[v]; ok {
+// findImportRecursiveRegexp does the same as `importRecursive` but uses a regexp
+// rather than parsing the AST of all files. This is much faster, but can lead to
+// false positives (e.g. if a string contains `import "foo"`).
+func findImportRecursiveRegexp(list map[string]bool, vm *jsonnet.VM, filename, content string) error {
+	matches := importsRegexp.FindAllStringSubmatch(content, -1)
+
+	for _, match := range matches {
+		importContents, foundAt, err := vm.ImportData(filename, match[2])
+		if err != nil {
+			continue
+		}
+		abs, _ := filepath.Abs(foundAt)
```

Comment:
Any reason why ignoring the error here? Is it guaranteed this will always succeed?